src/widget/systemtrayicon.h

```diff
@@ -48,6 +49,8 @@ class SystemTrayIcon : public QObject
 private:
     SystrayBackendType backendType;
     QSystemTrayIcon* qtIcon;
+    Widget *parent;
```

Comment:
Proposal: Rename to `mParent`. Thus, `this->parent = parent` would become `mParent = parent` in c'tor implementation [in cpp line 31](https://github.com/tux3/qTox/pull/2132/files#diff-1b85c272c0d41b3a9efecede73cfa60eR31).